### ISSUE TITLE
Replace tns-core-modules with @nativescript/core

### ISSUE THREAD

**Issue by fibric** [halfnelson/svelte-native#103]
Hi,

With the release of NativeScript 6.3, almost (if not all tns packages) are now available as [scoped packages](https://www.nativescript.org/blog/nativescript-6.3-the-holiday-edition#Scoped_Packages_Independent_from_the_Regular_Ones_13).

`tns-cope-modules` remains existing just for compatibility but the, devs [recommend switching](
https://github.com/NativeScript/NativeScript/tree/master/tns-core-modules-package) to `@nativescript/core`.

Edit:
This change is dependent on
- https://github.com/NativeScript/nativescript-dev-webpack

I think this issue should have low priority because `tns-core-modules` transparently resolves `@nativescript/core`.

**Comment by halfnelson** [comment]
I had a crack at this a couple of weeks ago and ran into the problem that nativescript-dev-webpack still had hard coded references to 'tns-core-modules', The migration wasn't too difficult and I will probably have another shot at it once the nativescript-dev-webpack changes land in production

**Comment by halfnelson** [comment]
Fixed in 0.8.0
